default/buildings.txt

```diff
@@ -625,7 +625,7 @@ BuildingType
             stackinggroup = "INDUSTRY_CENTER_STACK"
             effects = SetTargetIndustry value = Value + Target.Population * 3 * [[INDUSTRY_PER_POP]]
     ]
-    icon = ""
+    icon = "icons/tech/industrial_centre_ii.png"
```

Comment:
Ah, OK, never commit from a master branch, that makes some sense, I'll have to figure out the taskflow but I know what you're asking for now. I guess it's because I do most of my work on the fly while playing, just adding a quick fix to a problem, especially the icons or stringtable updates, I'll get there, thanks for the explanation.